src/components/Maps/AK_Fires.vue

```diff
@@ -72,13 +78,259 @@ export default {
           'abstract': 'This layer shows historical fire perimeters from 1940-2016.  _More recent wildfires often stop fires from spreading due to the lack of fuel, but does this always hold true?_\n\nTo access and learn more about this dataset, visit the [AICC](https://fire.ak.blm.gov).\n',
           'name': 'geonode:fireareahistory',
           'title': 'Historical extent, 1940-2016'
+        },
+        {
+          'name': 'fires_2017',
+          'title': 'All fires, 2017',
+          'object': () => {
+            console.log('in object thing, this: ', this)
+            console.log('in object thing, this.fireLayerGroup: ', this.fireLayerGroup)
+            return this.fireLayerGroup
+          },
+          'secondObject': () => { return this.secondFireLayerGroup },
+          'local': true,
+          'legend': false,
+          'abstract': '<img src="images/legend3.svg"/><p>This layer shows fires that occurred or are actively burning this year.</p><p>We update our map each hour from the source data available at the <a href="https://fire.ak.blm.gov" target="_blank" rel="externa">AICC</a> web site.</p><p><em>Where do most fires occur?  Where do most of the large fires occur?</em></p>'
+        }
+      ],
+      // Will initialize these in the created() method
+      activeFireIcon: undefined,
+      inactiveFireIcon: undefined,
+      firePolygons: undefined,
+      fireMarkers: undefined,
+      secondFirePolygons: undefined,
+      secondFireMarkers: undefined,
+      fireLayerGroup: undefined,
+      secondFireLayerGroup: undefined
+    }
+  },
+  created () {
+    let FireIcon = this.$L.Icon.extend({
+      options: {
+        iconUrl: '/static/active_fire.png',
+        iconSize: [30, 35],
+        shadowSize: [0, 0], // no shadow!
+        iconAnchor: [16, 34], // point of the icon which will correspond to marker's location
+        shadowAnchor: [0, 0],  // the same for the shadow
+        popupAnchor: [0, 0] // point from which the popup should open relative to the iconAnchor
+      }
+    })
+
+    this.activeFireIcon = new FireIcon()
+    this.inactiveFireIcon = new FireIcon({
+      iconUrl: '/static/inactive_fire.png'
+    })
+
+    // This will be the container for the fire markers & popups.
+    this.fireLayerGroup = this.$L.layerGroup()
+    this.secondFireLayerGroup = this.$L.layerGroup()
+
+    this.fetchFireData()
+  },
+  methods: {
+    getLocalLayers (layer) {
+      return {
+        first: this.fireLayerGroup,
+        second: this.secondFireLayerGroup
+      }
+    },
+    fetchFireData () {
+      return new Promise((resolve, reject) => {
+        if (this.firePolygons == null) {
```

Comment:
How strict are we being on using `===`? I've been defaulting to that style unless there's a reason not to. It's good practice, if linters are to be believed.

Reply:
Fixed to instead be `=== undefined`, to match the data initialization state.